src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -251,12 +251,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
             If Not PropertyControlData.IsSpecialValue(value) Then
 
-                Dim uIntValue As UInteger = 0
```

Comment:
I have handled this by making the interceptors to guard against unknown  values and return '0', which is old project system behavior. The value here will still be string but it will '0' and so property pages cannot crash